rmw_implementation/src/functions.cpp

```diff
@@ -127,6 +129,13 @@ get_symbol(const char * symbol_name)
   return lookup_symbol(get_library(), symbol_name);
 }
 
+void
+unload_library()
+{
+  g_symbol_rmw_init = nullptr;
```

Comment:
@cottsay hmm what about all the other symbols?

Reply:
I'll add them. Good call.